tests/integration/catalogue/test_options.py

```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase
+
+from oscar.test import factories
+
+
+class TestProductOptions(TestCase):
+
+    def setUp(self):
+        self.product = factories.create_product()
+        self.option = factories.OptionFactory()
+
+    def test_product_has_options(self):
```

Comment:
Need additional tests to check that `has_options` is set to false whenever all options are removed.